subprojects/core/src/integTest/groovy/org/gradle/api/ObjectExtensionServiceInjectionIntegrationTest.groovy

```diff
@@ -48,8 +46,8 @@ class ObjectExtensionServiceInjectionIntegrationTest extends AbstractIntegration
     }
 
     def "can inject service using getter"() {
-        buildFile << """
-            import ${Inject.name}
+        buildFile """
+            import javax.inject.Inject
```

Comment:
:x: `javax.inject.Inject` is now part of the default imports for scripts. If we change those lines, better simply remove the import

Reply:
Any idea why this isn't in the list of generated imparts I have now? What code should I invoke to get that list of included imports?

Reply:
Ah ... I guess because it looks like it is handled separately and in a dynamic way for the Groovy DSL:

https://github.com/gradle/gradle/blob/dea4e5e0ca40bcddc8515016c8d87c3d56743ad0/subprojects/core/src/main/java/org/gradle/groovy/scripts/internal/GradleResolveVisitor.java#L581

Reply:
Should be resolved. Made a change to GradleResolveVisitor that documented a static list of imports should be kept in sync.